delay.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+#Program to simulate network delay between nodes
+# Run as follows: "./tc_new.sh <instance_number>
```

Comment:
It's not run as `./delay.sh <instance_number>`?

Reply:
Good catch, changed the file name.